### PR TITLE
OKX-009 orchestrator dry-run recipe

### PR DESCRIPTION
**Pull request by pivox**
## Summary

OKX-009 adds the read-only/dry-run orchestrator recipe path for OKX without enabling any exchange writes.

- adds `recipe-r1-r16-okx-dry-run` fixture with OKX demo dry-run sets only
- adds `--target-exchange okx` to `runtime_recipe_runner.py` for R1/R2/R14
- runs `app:exchange:runtime-check okx perpetual` before OKX scenarios and exports `BLOCKED` without dispatch if `Schedule ready` is not `yes`
- keeps Fake/Paper as the default R1-R16 baseline and documents the OKX dry-run recipe/runbook

## Safety

- no mainnet write path
- no `dry_run=false` OKX recipe set
- no Bitmart fallback in the OKX recipe
- no secrets in fixtures/docs/tests
- R14 still validates the negative live guard before dispatch

## Tests

- `python3 -m pytest tests/test_runtime_recipe_runner.py tests/test_runtime_recipe_fixtures.py -q`
- `python3 -m pytest`
- `/Library/Frameworks/Python.framework/Versions/3.12/bin/python3 -m mkdocs build --strict`
- `git diff --check`

Related to #188.
Related to #234.
Part of OKX-009.

### CODE REVIEW

**Comment by pivox** [comment]
@codex please review this PR. Focus on correctness, regressions, runtime recipe safety, missing tests, and edge cases. Please post review threads on specific lines when useful, or a PR comment if the feedback is general.

File: python-orchestrator/scripts/runtime_recipe_runner.py [hunk 1]
```diff
@@ -167,10 +185,74 @@ def _check_orchestrator(self) -> tuple[bool, str]:
...
+        return parsed
+
     def _load_fixtures(self) -> dict[str, dict[str, Any]]:
         nominal = self._read_fixture("r1_r16_nominal_fake_dashboard.json")
         degraded = self._read_fixture("r1_r16_degraded_fake_dashboard.json")
-        return {"nominal": nominal, "degraded": degraded}
+        fixtures = {"nominal": nominal, "degraded": degraded}
+        okx_path = self.config.fixtures_dir / "r1_r16_okx_dry_run_dashboard.json"
+        if okx_path.exists():
+            fixtures["okx"] = json.loads(okx_path.read_text(encoding="utf-8"))
```
[the remark this answers is not in the document]
Comment (pivox) [reply python-orchestrator/scripts/runtime_recipe_runner.py]: Fixed in edfe82e: the runner now returns only the Fake/Paper fixtures unless `target_exchange` is `okx`; the OKX fixture is loaded and applied only after the OKX target path is selected. Added a regression assertion that the default Fake/Paper run does not create `recipe-r1-r16-okx-dry-run`.

[the remark this answers is not in the document]
Comment (pivox) [reply python-orchestrator/scripts/runtime_recipe_runner.py]: Fixed by the same edfe82e change: `_load_fixtures()` now returns immediately for non-OKX targets, so the default Fake/Paper recipe cannot load or apply `r1_r16_okx_dry_run_dashboard.json`. The regression test asserts the default run creates no `recipe-r1-r16-okx-dry-run` dashboard.

**Comment by pivox** [comment]
@codex review

File: python-orchestrator/scripts/runtime_recipe_runner.py [hunk 2]
```diff
@@ -119,18 +133,20 @@ def run(
         self.invocation_id = uuid4().hex[:12]
 
         service_ok, service_note = self._check_orchestrator()
+        preflight = self._run_target_preflight() if service_ok else {"status": "BLOCKED", "notes": service_note}
         dashboards: dict[str, dict[str, Any]] = {}
         results: list[dict[str, Any]] = []
 
-        if service_ok:
+        if service_ok and preflight["status"] == "PASS":
```
[the remark this answers is not in the document]
Comment (pivox) [reply python-orchestrator/scripts/runtime_recipe_runner.py]: Fixed in 983229c: OKX preflight failures now block only the OKX-targeted scenarios R1/R2/R14. The runner still applies Fake/Paper fixtures and can execute baseline scenarios such as R5 when OKX is not schedule-ready. Added a regression test for `target_exchange=okx` + failed OKX runtime-check + `R5`.

**Comment by pivox** [comment]
@codex review

File: python-orchestrator/fixtures/runtime-recipe/r1_r16_okx_dry_run_dashboard.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    "name": "recipe-r1-r16-okx-dry-run",
+    "enabled": true,
+    "description": "Recette runtime OKX dry-run - lecture/provider OKX et serialization locale, sans write exchange."
+  },
+  "sets": [
+    {
+      "set_id": "recipe_okx_regular",
+      "enabled": true,
+      "action": "mtf_run",
+      "exchange": "okx",
```
[the remark this answers is not in the document]
Comment (pivox) [reply python-orchestrator/fixtures/runtime-recipe/r1_r16_okx_dry_run_dashboard.json:17]: Fixed in a895634: the legacy `App\Contract\Runner\Dto\MtfRunnerRequestDto` now delegates exchange/market parsing to `ExchangeContextResolver`, matching the actual `/api/mtf/run` DTO path. Added a contract DTO regression test for trimmed uppercase OKX. Validation run includes the new contract DTO PHPUnit test, existing MTF runner DTO test, targeted PHPStan, lint:container, full python-orchestrator pytest, mkdocs strict, and git diff check.

**Comment by pivox** [comment]
@codex review

**Comment by pivox** [comment]
@codex review
